dvc/config.py

```diff
@@ -103,6 +103,7 @@ class RelPath(str):
 REMOTE_COMMON = {
     "url": str,
     "checksum_jobs": All(Coerce(int), Range(1)),
+    Optional("jobs", default=4): int,
```

Comment:
Why 4 though? Each tree has itsown default number of jobs. Won't this `default=4` always override it?

Reply:
I kept it close to the previous implementation, but more flexible. As I understand it now, the config-based value is not optional and must always be in the config. Is it?

Reply:
No, it should stay optional.